soccer/Robot.hpp

```diff
@@ -109,6 +113,25 @@ class Robot: public RobotPose
 		return *this == other;
 	}
 
+	std::string to_string() const {
```

Comment:
I'd still prefer it be camel case to match everything else.  And you're right, most things should have operator<< for easy use with stream-based logging.